README.md

```diff
@@ -1 +1,17 @@
-# opencost-grafana-dashboard
\ No newline at end of file
+# opencost-grafana-dashboard
+
+Collection of OpenCost Grafana dashboards.
+
+## Dashboards
+### alerting
+Currently no dashboards available. Contributions are welcome.
+
+### cost-reporter
+- opencost-cost-reporter-basic-overview-v0.0.1
+- opencost-cost-reporter-detailed-overview-v1.0.0
```

Comment:
Same comment, let's drop the version out of the name. The dashboard in the Helm repo didn't have a version.

Reply:
Same as this [comment](https://github.com/opencost/opencost-grafana-dashboard/pull/3#discussion_r1574373100)